cedar-policy/src/frontend/is_authorized.rs

```diff
@@ -370,51 +398,121 @@ fn constant_true() -> bool {
     true
 }
 
-fn parse_schema(
-    schema_json: Option<JsonValueWithNoDuplicateKeys>,
-) -> Result<Option<Schema>, Vec<String>> {
-    schema_json
-        .map(|v| Schema::from_json_value(v.into()))
-        .transpose()
-        .map_err(|e| vec![e.to_string()])
+#[derive(Debug, Serialize, Deserialize)]
+enum Schema {
+    /// Schema in Cedar's default, human-readable schema format. See <https://docs.cedarpolicy.com/schema/human-readable-schema.html>
```

Comment:
I thought we wanted to just call this `Cedar schema format` going forwards?

Reply:
Will change.